crates/biome_deserialize/src/json.rs

```diff
@@ -613,3 +616,78 @@ where
         deserialized: output,
     }
 }
+
+declare_node_union! {
+    AntJsonEntity =
+        JsonArrayValue
+        | JsonBogusValue
+        | JsonBooleanValue
+        | JsonNullValue
+        | JsonNumberValue
+        | JsonObjectValue
+        | JsonStringValue
+        | JsonMemberName
+}
```

Comment:
I think we _might_ not need this union anymore. Instead, I think we should receive a `JsonRoot` node instead.